### ISSUE TITLE
ampersand in link

### ISSUE THREAD

**Issue by kaniansky** [froala/wysiwyg-editor#4592]
##### Expected behavior.
Ampersandin a link is preserved and link is linkified

##### Actual behavior.
Ampersand is changed into HTML entity &amp; and only part of the link is linkified

##### Steps to reproduce the problem.
paste a link with ampersand (E.g.: https://www.google.com/search?q=froala+ampersand+bug&oq=bla&sourceid=chrome) from browser URL bar into Froala Editor

##### Editor version.
4.0.17 on https://froala.com/wysiwyg-editor/examples/iframe/

##### OS.
Linux Mint but seems to be also on Windows 11

##### Browser.
Tested on Chrome

##### Recording.
![image](https://user-images.githubusercontent.com/6592778/220032166-0db0211f-e6d4-4be3-ae3e-8ff317eabbb2.png)

**Comment by DarylBuckle** [comment]
I've been having the same issue after upgrading from 4.0.16 to 4.0.17. It wasn't an issue in 4.0.16. Also tried 4.0.18 and 4.0.19 and it's still a problem.

It's an annoying one for our users, and I can't roll back to 4.0.16 as we need the fix for backspace issue in iOS that's in 4.0.17. So I did a bit of digging and traced it back to this addition in 4.0.17 in both ```/js/froala_editor.pkgd.min.js``` and ```/js/plugins.pkgd.min.js```;

```js
&&!l.$el[0].getAttribute("plainpaste")
```

Deleting this condition from both files (which reverts this line back to how it was in 4.0.16) fixes this issue. Obviously, it's not ideal as no doubt this was put in to fix a different issue. But this issue seems like it's far more annoying than whatever issue this code was added in to fix.

Hopefully this helps anyone else who needs a workaround for the time being. Just find and replace both occurrences of the above text with empty string.
